website/pages/docs/k8s/connect/ingress-gateways.mdx

```diff
@@ -0,0 +1,207 @@
+---
+layout: docs
+page_title: Ingress Gateways - Kubernetes
+sidebar_title: Ingress Gateways
+description: Configuring Ingress Gateways on Kubernetes
+---
+
+# Ingress Gateways on Kubernetes
+
+-> 1.8.0+: This feature is available in Consul versions 1.8.0 and higher
+
+~> This topic requires familiarity with [Ingress Gateways](/docs/connect/ingress-gateway).
+
+This page describes how to enable external access to Connect service mesh services running inside Kubernetes using Consul ingress gateways.
+See [Ingress Gateways](/docs/connect/ingress-gateway) for more information on use-cases and how it works.
+
+Adding an ingress gateway is a multi-step process that consists of the following steps:
+
+* Setting the helm chart configuration
+* Deploying the helm chart
+* Configuring the gateway
+* Defining an Intention (if ACLs are enabled)
+* Deploying your application to Kubernetes
+* Connecting to your application
+
+## Setting the helm chart configuration
+When deploying the helm chart you must provide helm with a custom yaml file that contains your environment configuration.
+
+```yaml
+global:
+  name: consul
+connectInject:
+  enabled: true
+ingressGateways:
+  enabled: true
+  gateways:
+    - name: ingress-gateway
+      service:
+       type: LoadBalancer
+```
+~> *Note:* this will create a public unathenticated LoadBalancer in your cluster, please take appropriate security considerations.
```

Comment:
```suggestion
~> *Note:* this will create a public unauthenticated LoadBalancer in your cluster, please take appropriate security considerations.
```